app/javascript/dashboard/components/widgets/conversation/MessagesView.vue

```diff
@@ -52,6 +52,9 @@
       </button>
     </div>
     <ul class="conversation-panel">
+      <div class="action-button__wrap" @click="onToggleContactPanel">
```

Comment:
Might be better to have a button instead of a div with click listener, div will not be accessible.

Reply:
fixed https://github.com/chatwoot/chatwoot/pull/3507/commits/b3a06ee1cc4bc251a36e33969cf071bf352ca4a0